src/parser/tests/expression/ConditionalExpression.spec.ts

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable @typescript-eslint/no-for-in-array */
+import { expect } from 'chai';
+import { DiagnosticMessages } from '../../../DiagnosticMessages';
+import { TokenKind, Lexer } from '../../../lexer';
+import { Parser, ParseMode } from '../../Parser';
+import { token, EOF } from '../Parser.spec';
+import { BrsString, BrsBoolean } from '../../../brsTypes';
+import {AssignmentStatement, ExpressionStatement, ForEachStatement} from '../../Statement';
+import {
+    AALiteralExpression,
+    ArrayLiteralExpression,
+    CallExpression,
+    ConditionalExpression,
+    LiteralExpression
+} from "../../Expression";
+
+describe('parser conditional expressions', () => {
+    it('throws exception when used in brightscript scope', () => {
+        let { tokens } = Lexer.scan(`a = true ? "human" : "Zombie"`);
+        let { diagnostics } = Parser.parse(tokens, { mode: ParseMode.BrightScript });
+        expect(diagnostics[0]?.code).to.equal(DiagnosticMessages.bsFeatureNotSupportedInBrsFiles('').code);
```

Comment:
Can you change this to check the message instead? It makes diagnosing errors a little easier since you don't have to go look up the code.

Reply:
ok